spring-integration-debezium/src/test/java/org/springframework/integration/debezium/it/DebeziumStreamTests.java

```diff
@@ -47,14 +48,16 @@
 @DirtiesContext
 public class DebeziumStreamTests implements DebeziumMySqlTestContainer {
 
+	static final int EXPECTED_DB_TX_COUNT = 52;
```

Comment:
Can this constant go to that `DebeziumMySqlTestContainer` since it is really a part of that container environment.
I understand that we may have other tests where we add more transactions, but at lest this one is about initial DB setup.